src/module/actor/army/data.ts

```diff
@@ -0,0 +1,117 @@
+import { Alignment } from "@actor/creature/index.ts";
+import {
+    ActorAttributes,
+    ActorAttributesSource,
+    ActorDetails,
+    ActorDetailsSource,
+    ActorHitPoints,
+    ActorSystemData,
+    ActorSystemSource,
+    ActorTraitsData,
+    ActorTraitsSource,
+    BaseActorSourcePF2e,
+    BaseHitPointsSource,
+} from "@actor/data/base.ts";
+import { ARMY_TYPES } from "./values.ts";
+import { ActorSizePF2e } from "@actor/data/size.ts";
+
+type ArmySource = BaseActorSourcePF2e<"army", ArmySystemSource>;
+
+interface ArmySystemSource extends ActorSystemSource {
+    attributes: ArmyAttributesSource;
```

Comment:
Attribute is on the chopping block because of remaster, though I don't know what @stwlam plans to do yet.

Reply:
We still need to get everything out of there, so it's fine for now.